automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestDescriptions.scala

```diff
@@ -269,7 +269,7 @@ class TestDescriptions extends ExtractionTest {
 
   val t1c = "A convenient soil hydraulic property that will be used in this study is the matric flux potential " +
     "Mh0 (m2 d-1)"
-    passingTest should s"find descriptions from t1c: ${t1c}" taggedAs(Somebody) in {
+    failingTest should s"find descriptions from t1c: ${t1c}" taggedAs(Somebody) in {
```

Comment:
why did this test become failing?

Reply:
That was because I disallowed an adjective in a definition rule and potential was found as an adjective. I changed that back for now, so this is passing again.